chapter8.adoc

```diff
@@ -1,27 +1,66 @@
 [[chapter8]]
 [[Smsdedbg]]
-== `Smsdedbg`: Supervisor Domain External Debug
+== Supervisor Domain External Trace and Debug
 
-The <<MSDCFG>> CSR contains the `sdedbgalw` bit that controls whether the
-current scheduled SD is allowed to be external-debugged cite:[ExtDbg]. This bit
-is context switched (along with rest of the `msdcfg`) per SD.
+The <<MSDCFG>> CSR contains the `sdedbgalw` and `sdetrcalw` bits that control
+whether the current scheduled SD is allowed to be external-debugged or to be
+external-traced respectively.
```

Comment:
yes this extension related only to external debug and external trace interactions with supervisor domains - added a note in the introduction of this section.

Reply:
Your point extends the scope of the current trace spec very much. The trace control now is defined could either be accessed "internally" and "externally". They will be both affected by "halted" signal, thus your point "this extension related only to external debug and external trace interactions with supervisor domains" cannot be interpreted correctly if not otherwise explained. 
The idea of isolating "external" and "internal" must involve sufficient DTPM discussion and cannot easily made.